server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -5159,4 +5158,15 @@ public BucketResponse createBucketResponse(Bucket bucket) {
         populateAccount(bucketResponse, bucket.getAccountId());
         return bucketResponse;
     }
+
+    private String getDomainPath(String path){
+        StringBuilder domainPath = new StringBuilder();
+        if (StringUtils.isEmpty(path) || path.equals("/")){
+            domainPath.append("/");
+        }
+        else{
```

Comment:
@isupersky , @vishesh92 meant also to change 
```suggestion
        else {
```

Reply:
sorry, missed that. pushing the changes right away